package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "@auth0/nextjs-auth0": "^3.6.0",
     "@looker/sdk-node": "25.4.0",
+    "@mozilla/nimbus-schemas": "^3001.0.0",
     "@mozilla/nimbus-shared": "^2.5.2",
```

Comment:
From reading https://github.com/mozilla/nimbus-shared, it sounds like we don't need `nimbus-shared` any more.